sources/packages/backend/apps/api/src/services/disbursement-schedule/disbursement-schedule-service.ts

```diff
@@ -67,7 +67,8 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
       .andWhere("location.id = :locationId", { locationId })
       .andWhere("application.applicationStatus IN (:...status)", {
         status: [ApplicationStatus.Enrolment, ApplicationStatus.Completed],
-      });
+      })
```

Comment:
There are 2 more places where the condition needs to be added. 

1.Load application details for COE.

Reply:
There is also a ministry API to confirm enrolment in addition to institution endpoint. Both the endpoints must be validated.